tests/k6/scenarios/core2_qa_control_plane_ops.js

```diff
@@ -143,9 +178,21 @@ function handleCtlOp(config, op, modelTypeIx, existingModels) {
         case seldonOpType.CREATE:
         case seldonOpType.UPDATE:
             opOk = k8s.loadModel(modelName, modelCRYaml, true)
+            if (config.isLoadPipeline) {
+                opOk = k8s.loadPipeline(generatePipelineName(modelName), pipelineCRYaml, true) && opOk
+            }
             break;
         case seldonOpType.DELETE:
             opOk = k8s.unloadModel(modelName, true)
+            if (config.isLoadPipeline) {
+                // a model can go away while a pipeline is still loaded, we then simulate this behavior
+                // by not unloading the pipeline in 50% of the cases
+                // TODO: make it an environment variable?
+                let unloadPipeline = Math.random() < 0.5 ? 0 : 1
+                if (unloadPipeline) {
+                    opOk = k8s.unloadPipeline(generatePipelineName(modelName), true) && opOk
```

Comment:
opOk is not bool but seldonOpExecStatus

Reply:
fixed in https://github.com/SeldonIO/seldon-core/pull/5636/commits/afbdd3b3a3ec358ce02c4198e14e191b78e0b1cc